engines/cine/anim.cpp

```diff
@@ -798,8 +816,34 @@ int loadSeq(const char *resourceName, int16 idx) {
  */
 int loadResource(const char *resourceName, int16 idx, int16 frameIndex) {
 	int result = -1; // Return an error by default
+
+	if (g_cine->getGameType() == Cine::GType_OS &&
+		g_cine->getPlatform() == Common::kPlatformDOS &&
+		g_sound->musicType() != MT_MT32 &&
+		(strstr(resourceName, ".SPL") || strstr(resourceName, ".H32")))
+	{
```

Comment:
The code formatting is off. The parentheses must stick to the previous line.

Reply:
Taken into account.